app/src/main/res/layout/welcome.xml

```diff
@@ -42,13 +42,28 @@
       app:layout_constraintBottom_toBottomOf="@id/welcome_text_view"
       app:layout_constraintTop_toTopOf="@+id/welcome_text_view" />
 
+    <TextView
+      android:id="@+id/profile_name_textview2"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:ellipsize="end"
+      android:fontFamily="sans-serif"
+      android:maxLines="1"
+      android:text="@{String.format(@string/welcome_profile_name, viewModel.profileName)}"
+      android:textColor="@color/oppiaPrimaryText"
+      android:textSize="24sp"
+      android:visibility="gone"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="@+id/welcome_text_view"
+      app:layout_constraintTop_toBottomOf="@+id/welcome_text_view" />
+
     <View
       android:layout_width="0dp"
       android:layout_height="6dp"
       android:layout_marginTop="4dp"
       android:background="@color/oppiaLightBrown"
       app:layout_constraintEnd_toEndOf="@+id/welcome_text_view"
       app:layout_constraintStart_toStartOf="@+id/welcome_text_view"
-      app:layout_constraintTop_toBottomOf="@+id/welcome_text_view" />
+      app:layout_constraintTop_toBottomOf="@+id/profile_name_textview2" />
   </androidx.constraintlayout.widget.ConstraintLayout>
-</layout>
+</layout>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
@rt4914 I will add EOF

Reply:
The orange line was intended to go under the greeting, but seeing your example, it seems best suited under the textbox that holds both the greeting and name. So this would mean that the orange line should always be positioned below the textbox on its left-hand side.

Reply:
@mschanteltc Thanks.

@coder2699 Please add EOF.

Reply:
@rt4914 I have added EOF